README.md

```diff
@@ -157,17 +151,19 @@ The `vue-component` directive provides three main attributes:
 <vue-component vprops-first-name="ctrl.person.firstName" vprops-last-name="ctrl.person.lastName" />
 ```
 
-- `watch-depth` attribute indicates which watch strategy to detect the changes of the scope. The possible values as follows:
+- `watch-depth` attribute indicates which watch strategy to detect the changes on Angular Scope objects. The possible values as follows:
 
   | value                 | description                              |
   | --------------------- | ---------------------------------------- |
-  | reference *(default)* | watches the object reference             |
-  | collection            | same as angular `$watchCollection`, shallow watches the properties of the object: for arrays, it watches the array items; for object maps, it watches the properties |
-  | value                 | deep watches every property inside the object |
+  | reference | *(default)* watches the object reference             |
+  | collection            | *(rarely used)* same as angular `$watchCollection`, shallow watches the properties of the object: for arrays, it watches the array items; for object maps, it watches the properties |
+  | value                 | *(rarely used)* deep watches every property inside the object |
 
-**NOTE** the `value` strategy is not recommended because Angular copies the entire object and traverses every property insides to detect the changes in each digest cycle and it therefore causes a heavy computation.
+**NOTES**
 
-### the `createVueComponent` factory
+- The `value` strategy is **not recommended** and it causes a heavy computation. To detect the change, Angular copies the entire object and traverses every property insidess in each digest cycle.
```

Comment:
`and traverses every property in each digest cycle.` Looks like `insidess` a typo ;-)